easybuild/easyconfigs/s/Salmon/Salmon-1.2.0-gompi-2019b.eb

```diff
@@ -0,0 +1,49 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# Based on Robert Qiao <rob.qiao@flinders.edu.au> contribution
+# #
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '1.2.0'
+
+homepage = 'https://salmon.readthedocs.io'
+description = """Salmon is a wicked-fast program to produce a highly-accurate,
+transcript-level quantification estimates from RNA-seq data. Salmon achieves
+its accuracy and speed via a number of different innovations, including the use
+of selective-alignment, and massively-parallel stochastic collapsed variational
+inference."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+github_account = 'COMBINE-lab'
+source_urls = ['https://github.com/%(github_account)s/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['c8a6647484ff630e4958aa4540cba28c0913b26922313876d747aba65eefb96f']
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('pkg-config', '0.29.2'),
+    ('jemalloc', '5.2.1'),
+    ('Boost', '1.71.0'),
```

Comment:
Could you please clarify why Boost has been moved from a runtime dependency to a build dependency?

Reply:
It is built into static mode

Reply:
I was not aware that Salmon only built static libs. :+1: